pkg/kubectl/kubectl.go

```diff
@@ -43,6 +47,40 @@ func makeImageList(spec *api.PodSpec) string {
 	return strings.Join(listOfImages(spec), ",")
 }
 
+type ThirdPartyResourceMapper struct {
```

Comment:
How does this relate to the existing `thirdPartyResourceDataMapper`?  It seems like they're both RESTMappers.